### DISCUSSION TITLE
Highlight specific words in the title and abstract

### DISCUSSION THREAD

**Discussion by tomseinen** [asreview/asreview#995] [Show and tell]
## Feature Request

**Problem**
It can be very useful to know which terms from your systematic review search query/strategy are mentioned in the title and abstract of the articles you are reviewing. Often, I need to read the entire abstract to find out which of the terms are mentioned in the text and whether they are mentioned in the correct context.

Being able to see the words that you used in your search query (or other custom words) at a glance speeds up the review process significantly. I tested it with an alternative solution and it works great.

**Solution**
A possibility to input your custom terms that will be highlighted in the title and abstract.
Make this input field available through an extra button with a [highlighter icon](https://fontawesome.com/icons/highlighter?style=solid) next to the export and settings buttons.
Add an on-off switch to show or hide the highlights.

**Alternative solution**
I currently use the Chrome extension [Multi-highlight](https://chrome.google.com/webstore/detail/multi-highlight/pfgfgjlejbbpfmcfjhdmikihihddeeji?hl=en) to highlight specific terms on the webpage in random colours.
One disadvantage is that it is also activated on all the other browser windows.
Example of use:
![example](https://user-images.githubusercontent.com/54170403/105996759-0be28200-60ab-11eb-95db-ac1ecfdabfb7.PNG)

**Teachability, Documentation, Adoption, Migration Strategy**
Easy to teach, just enter the custom terms that you want to highlight.
This feature is only added to the UI and does not cause any structural changes to the data or methods.

Optionally include the option for adding regular expressions, but that is maybe a bit too much.

What do you think?

Thank you for the excellent tool,
Tom

**Comment by bmkramer** [comment]
Thanks for your interest in ASReview! 

Others can comment better on the technical possibilities. From a conceptual perspective, we actively try to avoid including information in ASReview that can lead to biases in selection - which, for example,  is the reason no author names or journal titles are shown. It would be interesting to consider to what extent selecting terms to highlight might run a risk of introducing bias (e.g. when selection is consciously or unconsciously influenced by the number or location of highlighted terms).

**Reply by Cathalijn** [reply]
While I think it is great that ASReview prevents selection bias by blinding authors and journals, I am not so concerned with bias due to highlighting; we use the highlights to find the relevant parts of the abstracts to speed up screening, when the highlighted phrases do not allow for a conclusive decision, we read everything.

**Comment by Cathalijn** [comment]
I really like the idea of using a chrome plug-in, but the link does not seem to work....

**Reply by Rensvandeschoot** [reply]
Can it be this one? https://chrome.google.com/webstore/detail/multi-highlight/ghinpogonpmlphfkmnjfapdaoaainejn

**Reply by Cathalijn** [reply]
It is :-) Thanks!

**Reply by tomseinen** [reply]
Ah yes, it seems like the extension I posted has been removed. This is also an alternative: https://chrome.google.com/webstore/detail/highlighty-search-find-mu/migdhldfbapmodfbmgpofnikfbfpbbon

**Reply by tomseinen** [reply]
This one might even be better: https://chrome.google.com/webstore/detail/highlight-this-finds-and/fgmbnmjmbjenlhbefngfibmjkpbcljaj/related

**Reply by amit-soni7** [reply]
best i found is https://chromewebstore.google.com/detail/highlight-magic/ebjdbdfklikkclechjijionbjkhhljml
in this we could make or import list based on PICO/inclusion or exclusion criteria and color code them too.

**Reply by sagebiej** [reply]
a bit redundant maybe, but I also wrote an extension for exactly this purpose. [https://github.com/sagebiej/asreview-keyword-highlighter ](https://github.com/sagebiej/asreview-keyword-highlighter )
